src/newrelic.ts

```diff
@@ -1,12 +1,11 @@
-/* Copyright © 2021-2022 Seneca Project Contributors, MIT License. */
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
 
 
-import NewRelic from 'newrelic'
-
-type NewRelicOptions = {};
-
-type Spec = Record<any, any>
-
+import NewRelic from 'newrelic';
```

Comment:
separate imports into three groups (separated by whitespace):
- built in modules (e.g util)
- external modules+types (e.g newrelic)
- internal modules+types (e.g. ./types)